mne/chpi.py

```diff
@@ -149,6 +150,115 @@ def head_pos_to_trans_rot_t(quats):
     return translation, rotation, t
 
 
+def _apply_quat(quat, pts, move=True):
+    """Apply MaxFilter-formatted head position parameters to points."""
+    trans = np.concatenate(
+        (quat_to_rot(quat[:3]),
+         quat[3:][:, np.newaxis]), axis=1)
+
+    return(apply_trans(trans, pts, move=move))
+
+
+def extract_head_pos_ctf(raw, gof_limit=0.98):
```

Comment:
... and we've probably talked about this before, but maybe we should just have one `calculate_head_positions(raw)` function. If you pass a CTF raw, it uses this code. If you pass Neuromag raw, it uses the Neuromag code. WDYT?

If you think we should do it, but do it in a separate PR, then let's make this function private for now.